### PR TITLE
Add aarch64 and ppc64le architectures

### PR DESCRIPTION
**Pull request by ehfd**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   skip: true  # [not linux]
-  number: 0
+  number: 1
```
Comment (isuruf) [inline recipe/meta.yaml]: You don't need to bump this.

**Comment by traversaro** [comment]
@ehfd do you want to add yourself to the maintainers?

**Comment by ehfd** [comment]
We are all in the maintainers.

File: recipe/meta.yaml (hunk 1 above)
Comment (ehfd) [reply recipe/meta.yaml]: Reverted.

**Comment by ehfd** [comment]
@isuruf No CI is running.

Edit: works.

**Comment by traversaro** [comment]
> We are all in the maintainers.

Sorry, my bad, I misread the maintainer list, thanks again for all the work!

**Pull request merged**